.github/workflows/build_orca.yml

```diff
@@ -339,7 +339,7 @@ jobs:
         working-directory: ${{ github.workspace }}/build/src
         shell: bash
         run: |
-          ./OrcaSlicer_profile_validator -p ${{ github.workspace }}/resources/profiles -g 1
+          ./build/src/OrcaSlicer_profile_validator -p ${{ github.workspace }}/resources/profiles -g 1
```

Comment:
This won’t make any difference since we already set the working directory to "build/src" two lines above.  
The executable is now located at `${{ github.workspace }}/build/src/Release`.
Let's update the `working-directory: ${{ github.workspace }}/build/src` to `working-directory: ${{ github.workspace }}/build/src/Release`

Reply:
;)